VMEncryption/main/handle.py

```diff
@@ -270,6 +272,196 @@ def get_protected_settings():
     else:
         return protected_settings_str
 
+def create_temp_file(file_content):
```

Comment:
Can we move these new functions to a new file called something like cvmLuksUtil?